pkg/controller/snapshot_delete_test.go

```diff
@@ -128,10 +129,21 @@ func TestDeleteSync(t *testing.T) {
 			expectedDeleteCalls: []deleteCall{{"sid1-1", nil, nil}},
 			test:                testSyncContent,
 		},
+		{
+			name:                "1-1 - content with retain policy will not be deleted if it is bound to a non-exist snapshot and also has a snapshot uid specified",
```

Comment:
the number of test case is the same as bellow, we can put this and "2-1" case after the "1-11" case and rename it "1-12"

Reply:
fixed